ambuda/seed/texts/gretil.py

```diff
@@ -70,161 +61,45 @@ def fetch_latest_data():
     subprocess.call("git reset --hard origin/main", shell=True, cwd=DATA_DIR)
 
 
-@dataclass
-class Block:
-    slug: str
-    #: XML blob.
-    blob: str
-
-
-@dataclass
-class Section:
-    slug: str
-    blocks: list[Block]
-
+def _create_new_text(session, spec: Spec, document: Document):
+    text = db.Text(slug=spec.slug, title=spec.title, header=document.header)
+    session.add(text)
+    session.flush()
 
-@dataclass
-class Document:
-    #: <teiHeader> XML blob.
-    header: str
-    sections: list[Section]
-
-
-def log(*a):
-    print(*a)
-
-
-def remove_namespace(xml: ET.Element, prefix: str):
-    """Remove the given namespace prefix from all elements.
-
-    ElementTree expands tidy namespaced names like "xml:id" into names like
-    "{http://www.w3.org/XML/1998/namespace}id", which are less usable. This
-    function removes these namespaces so that downstream code can be more
-    readable.
-    """
-    for el in xml.iter("*"):
-        if el.tag.startswith(prefix):
-            el.tag = el.tag[len(prefix) :]
-
-
-def to_slp1(xml: ET.Element):
-    """Transliterate inline elements."""
-    t = sanscript.transliterate
-    for el in xml.iter("*"):
-        if el.text:
-            el.text = t(el.text, sanscript.IAST, sanscript.DEVANAGARI)
-        if el.tail:
-            el.tail = t(el.tail, sanscript.IAST, sanscript.DEVANAGARI)
-
-
-def delete_unused_elements(xml: ET.Element):
-    """Remove unused elements in-place."""
-    for L in xml.iter("l"):
-        for el in L:
-            # Delete tag, keep text
-            if el.tag in {"seg", "hi"}:
-                el.tag = None
-            # Delete tag and text.
-            if el.tag in {"note"}:
-                el.tag = None
-                el.clear()
-        text = "".join(L.itertext())
-        text = text.replace("-", "")
-        L.clear()
-        L.text = text
-
-
-def _make_section(xml: ET.Element, section_slug: str) -> Section:
-    section = Section(slug=section_slug, blocks=[])
     n = 1
```

Comment:
Now that we're touching these files, can we rename this to `block_number` or `number_of_blocks`?